bin/scavenger-checker-all

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+#sinfo -M industry -p scavenger --states=idle --format="I %n %Y %m %f"|tail -n+3
+#sinfo -M chemistry -p scavenger --states=idle --format="C %n %Y %m %f"|tail -n+3
+#sinfo -M mae -p scavenger --states=idle --format="M %n %Y %m %f"|tail -n+3
+#sinfo -M physics -p scavenger --states=idle --format="P %n %Y %m %f"|tail -n+3
```

Comment:
chemistry, mae, & physics should be removed and faculty added

Reply:
Done. Fixed in 7134b42602c4569f1df3a87d35d4a694e336b03d